src/account/account.ts

```diff
@@ -144,7 +150,26 @@ export function getAccountAddressAsString(addressEncodedInB64: string): string {
  * @returns The account information
  */
 export async function getAccountInformation(sender: string | SendTransactionFrom, algod: Algodv2): Promise<AccountInformation> {
-  return new AccountManager(new ClientManager({ algod })).getInformation(getSenderAddress(sender))
+  const account = AccountInformationModel.from_obj_for_encoding(await algod.accountInformation(getSenderAddress(sender)).do())
```

Comment:
Keeping backwards compatibility in the legacy method (made changes to this method in `AccountManager` to return better types (`AlgoAmount` rather than `number`).